doc/plutus-core-spec/flat-serialisation.tex

```diff
@@ -195,16 +195,17 @@ \subsection{Arrays}
 
 \subsection{Natural numbers}
 We encode natural numbers by splitting their binary representations into
-sequences of 7-bit blocks, then emitting these as a list with the \textbf{least
-  significant block first}:
+sequences of 7-bit blocks, then emitting all but last chunks as a list with
+the \textbf{least significant block first}. The last chunk(most significant chunk)
+will be encoded as regular 7-bit block and appended right after the list of chunks.
```

Comment:
You could maybe add a sentence saying that this amounts to all of the leading blocks being prefixed by 1 and the final one by 0.  That seems a bit easier to grasp (but that might just be me).